src/inference/dev_api/openvino/runtime/threading/istreams_executor.hpp

```diff
@@ -150,6 +150,12 @@ class OPENVINO_RUNTIME_API IStreamsExecutor : virtual public ITaskExecutor {
               _threadPreferredCoreType(threadPreferredCoreType),
               _streams_info_table{streamsInfoTable},
               _cpu_reservation{cpuReservation} {}
+
```

Comment:
Please add a description of this method. It's not obvious how exactly the config is updated

Reply:
updated